packages/performance/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
 
+## [1.2.3] - 2020-9-16
+
+### Changed
```

Comment:
```suggestion
### Added
```

Exposing a method that already existed could be counted as adding of new functionality.